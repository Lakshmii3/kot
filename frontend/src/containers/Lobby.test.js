import React from 'react';
import { shallow } from 'enzyme';

import LoginLayout from './Lobby';

describe('Verify lobby template', () => {
  it('Lobby should render correctly', () => {
    const errorPage = shallow(<LoginLayout />);
    expect(errorPage).toMatchSnapshot();
  });
})