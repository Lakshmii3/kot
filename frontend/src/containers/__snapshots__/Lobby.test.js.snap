// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Verify lobby template Lobby should render correctly 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-sm"
    />
    <div
      className="col-sm"
    >
      Your In the Lobby...
    </div>
    <div
      className="col-sm"
    />
  </div>
</div>
`;
